ci-operator/jobs/redhat-developer/jenkins-operator/OWNERS

```diff
@@ -0,0 +1,18 @@
+approvers:
+- akram
+- waveywaves
+- sbose78
+- otaviof
+- gabemontero
+- bparees
```

Comment:
i don't need to be here, i'm in the toplevle jobs owners file

Reply:
okay, I will remove you from this list then